includes/Modules/Idea_Hub.php

```diff
@@ -146,12 +161,118 @@ function( $maybe_empty, $postarr ) {
+			'idea-hub_saved-ideas',
+			array(
+				'content'         => function() {
+					ob_start();
+					?>
+					<p>
+						<?php echo esc_html( esc_html__( 'Need some inspiration? Revisit your saved ideas in Site Kit', 'google-site-kit' ) ); ?>
+						<a href="<?php echo esc_url( $this->context->admin_url() . '#saved-ideas' ); ?>"
+						><?php echo esc_html( esc_html__( 'See saved ideas', 'google-site-kit' ) ); ?></a>
+					</p>
+					<?php
+					return ob_get_clean();
+				},
+				'type'            => Notice::TYPE_INFO,
+				'active_callback' => function() use ( $transients, $dismissed_items ) {
+					$saved_ideas = $transients->get( self::TRANSIENT_SAVED_IDEAS );
+					if ( false === $saved_ideas ) {
+						$saved_ideas = $this->get_data( 'saved-ideas' );
+						$transients->set( self::TRANSIENT_SAVED_IDEAS, $saved_ideas, DAY_IN_SECONDS );
+					}
+					$has_saved_ideas = count( $saved_ideas ) > 0;
+					if ( ! $has_saved_ideas && $dismissed_items->is_dismissed( 'saved-ideas' ) ) {
+						// Saved items no longer need to be dismissed as there are none currently.
+						$dismissed_items->delete( 'saved-ideas' );
+					}
+					if ( $dismissed_items->is_dismissed( 'saved-ideas' ) ) {
+						return false;
+					}
+
+					return $has_saved_ideas;
+				},
+				'dismissible'     => true,
+			)
+		);
+		$notices[] = new Notice(
+			'idea-hub_new-ideas',
+			array(
+				'content'         => function() {
+					ob_start();
+					?>
+					<p>
+						<?php echo esc_html( esc_html__( 'Need some inspiration? Here are some new ideas from Site Kit’s Idea Hub', 'google-site-kit' ) ); ?>
+						<a href="<?php echo esc_url( $this->context->admin_url() . '#new-ideas' ); ?>"
+						><?php echo esc_html( esc_html__( 'See new ideas', 'google-site-kit' ) ); ?></a>
+					</p>
+					<?php
+					return ob_get_clean();
+				},
+				'type'            => Notice::TYPE_INFO,
+				'active_callback' => function() use ( $transients, $dismissed_items ) {
+					if ( $dismissed_items->is_dismissed( 'new-ideas' ) || $dismissed_items->is_dismissed( 'saved-ideas' ) ) {
+						return false;
+					}
+					$saved_ideas = $transients->get( self::TRANSIENT_SAVED_IDEAS );
+					if ( false === $saved_ideas ) {
+						$saved_ideas = $this->get_data( 'saved-ideas' );
+						$transients->set( self::TRANSIENT_SAVED_IDEAS, $saved_ideas, DAY_IN_SECONDS );
+					}
+					$has_saved_ideas = count( $saved_ideas ) > 0;
+
+					if ( $has_saved_ideas ) {
+						// Don't show new ideas notice if there are saved ideas,
+						// irrespective of whether we show them the saved ideas notice.
+						return false;
+					}
+
+					$new_ideas = $transients->get( self::TRANSIENT_NEW_IDEAS );
+					if ( false === $new_ideas ) {
+						$new_ideas = $this->get_data( 'new-ideas' );
+						$transients->set( self::TRANSIENT_NEW_IDEAS, $new_ideas, DAY_IN_SECONDS );
+					}
+
+					$has_new_ideas = count( $new_ideas ) > 0;
+
+					return $has_new_ideas && ! $dismissed_items->is_dismissed( 'new-ideas' );
```

Comment:
No need to check whether the `new-ideas` notice is dismissed here because we already checked it above.
```suggestion
					return $has_new_ideas;
```

Reply:
https://github.com/google/site-kit-wp/pull/3668/commits/c5ab2b69e7d1b6bb5244ed52f8a5f22c071235c6

Reply:
https://github.com/google/site-kit-wp/pull/3668/commits/c5ab2b69e7d1b6bb5244ed52f8a5f22c071235c6